en/micro-integrator/docs/setup/brokers/configure-with-azureservicebus.md

```diff
@@ -0,0 +1,57 @@
+# Connecting to Azure Service Bus
+
+This section describes how to configure WSO2 Micro Integrator to connect with Azure Service Bus.
+
+The integration between this message broker and WSO2 Micro Integrator is straightforward. The guaranteed delivery integration pattern must be used to implement the mediation flow.
```

Comment:
We should say something like this.
The Azure Service Bus complies with Both AMQP 1.0 and JMS 2.0 [1]. The Micro Integrator uses its inbuilt JMS transport to send and receive messages from the Azure Service Bus. The configurations are similar to connecting with other JMS brokers.

[1] https://docs.microsoft.com/en-us/azure/service-bus-messaging/service-bus-messaging-overview#compliance-with-standards-and-protocols

Reply:
Added this in.